src/components/project/ProjectActions.tsx

```diff
@@ -26,7 +30,7 @@ const ProjectActions: React.FC<ProjectActionsProps> = ({ project }) => {
         <PublishButton project={project} disabled={disabled} />
       </Box>
       <Box flex={1}>
-        <DeleteButton project={project} />
+        <DeleteButton project={project} disabled={disable} />
```

Comment:
I noticed here that with this button, you use `disabled={disable}`, but on others you use `disabled={disabled}`. Is this intentional?